scripts/gui_main_mpl.py

```diff
@@ -0,0 +1,200 @@
+import tkinter as tk
```

Comment:
this script is a continuation of the gui_main script and probably will end up as the main script for gui use; need to rewrite graph trajectory use to incorporate the tkinter-matplotlib connection for cleaner graphing in GUI